sdk/batch/azure-batch/azure/batch/models/_patch.py

```diff
@@ -0,0 +1,76 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""Customize generated code here.
+
+Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
+"""
+from typing import List, Any
+from ._models import BatchPoolReplaceContent as BatchPoolReplaceContentGenerated
+from .._model_base import rest_field
+
+__all__: List[str] = [
+    "BatchPoolReplaceContent",
+    "CreateTasksErrorException",
+]  # Add all objects you want publicly available to users at this package level
+
+class BatchPoolReplaceContent(BatchPoolReplaceContentGenerated):
+    certificate_references: List[str] = rest_field(name="certificateReferences")
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:  # pylint: disable=useless-super-delegation
+        super().__init__(*args, **kwargs)
+        self.certificate_references = []
+
+
+class CreateTasksErrorException(Exception):
+    """Aggregate Exception containing details for any failures from a task add operation.
+
+    :param str message: Error message describing exit reason
+    :param [~TaskAddParameter] pending_tasks: List of tasks remaining to be submitted.
+    :param [~TaskAddResult] failure_tasks: List of tasks which failed to add
+    :param [~Exception] errors: List of unknown errors forcing early termination
+    """
+
+    def __init__(self, pending_tasks=None, failure_tasks=None, errors=None):
```

Comment:
Can we make these parameters keyword-only as well?

Reply:
Would prefer to keep them as positional arguments since they are needed to determine the error message.